index.js

```diff
@@ -1,9 +1,7 @@
 'use strict';
 const deburr = require('lodash.deburr');
 
-module.exports = string => {
-	const separator = '-';
-
+module.exports = (string, separator = '-') => {
```

Comment:
It's not as easy as this as you're passing it to regexes, so you need to escape it with https://github.com/sindresorhus/escape-string-regexp

It should also be an options-object instead of normal argument to allow for future additional options.